static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -44,7 +46,11 @@ define([
     };
 
     MostPopular.prototype.prerender = function () {
-        if (!(config.page.shouldHideAdverts || this.mobileMaximumSlotsReached())) {
+        if (!(
+            config.page.shouldHideAdverts || /* sensitive pages */
```

Comment:
I know that you didn't write it originally but we should also test if the page is one of the children's book site (config.page.section === 'childrens-books-site'). I don't know why it is not marked as a sensitive one but ads are not allowed there as well.